sassutils/wsgi.py

```diff
@@ -143,7 +143,9 @@ def __call__(self, environ, start_response):
                         b'/*\n', str(e).encode('utf-8'), b'\n*/\n\n',
                         b'body:before { content: ',
                         self.quote_css_string(str(e)).encode('utf-8'),
-                        b'; color: maroon; background-color: white; }'
+                        b'; color: maroon; background-color: white',
+                        b'; white-space: pre-wrap; display: block; ',
+                        b'font-family: monospace, "Courier New"; user-select: text; }'
```

Comment:
If `monospace` is prior to `Courier New` the latter would do nothing.  Is it by your intention?